VoteCounter/src/GUI.cs

```diff
@@ -27,10 +30,55 @@ private void btnGenerateVoteCount_Click(object sender, EventArgs e)
 
             if (Int32.TryParse(txtStartingPost.Text, out startingPostNumber) && Int32.TryParse(txtEndingPost.Text, out endingPostNumber))
             {
+                //make a List
+                List<Player> players = new List<Player>();
+
+                //set the players
+                foreach (DataGridViewRow row in listPlayers.Rows)
+                {
+                    Player newPlayer = new Player("");
+
+                    foreach (DataGridViewColumn column in listPlayers.Columns)
```

Comment:
What strings?

Also maybe we can do that. I am unfamiliar with that specific syntax.